vanilla-framework/vanilla-docs-theme
# Update vanilla-framework to v1.4.2

## Done
Update vanilla-framework to the latest release

## QA
- Run `./run clean`
- Run `./run`
- Go to http://0.0.0.0:8103/vanilla-docs-theme/
- Check the patterns look ok
_This is why it's important to create examples of all patterns_
